stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -247,7 +274,8 @@ void populateVhloToStablehloPatterns(RewritePatternSet* patterns,
   populateVhloToStablehloPatterns<
 #define GET_OP_LIST
 #include "stablehlo/dialect/StablehloOps.cpp.inc"
-      >(patterns, converter, context);
+      , func::CallOp, func::FuncOp>(patterns, converter, context);
```

Comment:
Why do we need to handle `func::CallOp` and `func::FuncOp` here at all?

Reply:
Since we create patterns based on Mapping results, these two create the following patterns:

```
vhlo.func --> func.func
vhlo.call --> func.call
```

When these ops are added to StableHLO (and therefore included by `GET_OP_LIST`) we can remove these.